stubs/toposort/toposort.pyi

```diff
@@ -0,0 +1,10 @@
+from typing import Any, Iterator, TypeVar
+
+_T = TypeVar("_T")
+
+class CircularDependencyError(ValueError):
+    data: dict[Any, set[Any]]
+    def __init__(self, data: dict[Any, set[Any]]) -> None: ...
```

Comment:
I initially made this generic, but then I realized that there's no good way to catch the exception properly. For example, there's no way to say "this function call raises `CircularDependencyError[str]`", which would be needed to get something meaningful from this:

```
foo: dict[str, set[str]] = {"a": {"b"}, "b": {"a"}}
try:
    print(list(toposort.toposort(foo)))
except toposort.CircularDependencyError as e:
    reveal_type(e.data)
```